Styles/header.css

```diff
@@ -0,0 +1,53 @@
+.nav-bar {
+  background: transparent;
+  color: var(--text-color);
+  padding: 0;
+}
+
+.nav-logo {
+  margin: 1rem;
+}
+
+.nav-search {
+  min-width: 30%;
+}
+
+.search-container {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border: 2px solid black;
+  border-radius: 8px;
+  padding: 0.5rem;
+}
+
+.search-container > * {
+  background-color: transparent;
+  border: none;
+}
+
+.search-container input {
+  outline: transparent;
```

Comment:
It's  width can be increased so it takes up full width. :)

Reply:
thanks for pointing out, made the suggested change.